### ISSUE TITLE
How to implement children routes?

### ISSUE THREAD

**Issue by caesar1shi** [nuejs/nue#157]
For instance, if the initial link is http://localhost:8080/video/cartoon/:movieId, we need to first route to '/video' to mount the video page, and then route to '/video/cartoon/:movieId' to mount the cartoon subpage.

**Comment by tipiirai** [comment]
Good point. How is this implemented currently on the (simplest possible) routing alternative? Maybe mimic that API.
